Code/GraphMol/FileParsers/MolSupplier.h

```diff
@@ -15,10 +15,18 @@
 
 #include <string>
 #include <list>
+#include <memory>
 #include <vector>
 #include <iostream>
 #include <GraphMol/ROMol.h>
 
+namespace schrodinger {
```

Comment:
Should this be in an #ifdef BUILD_COORDGEN_SUPPORT block?

Reply:
I'll put the `#ifdef` in here so it's clear why this code is here, but the forward definition is harmless if unused (won't cause missing symbol problems on link).